src/main/java/de/uni_mannheim/informatik/dws/wdi/WDI_Project_T2/IdentityResolution/IR_using_linear_combination.java

```diff
@@ -0,0 +1,194 @@
+package de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.IdentityResolution;
+
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.blocking.CarBlockingKeyByManufacturerGenerator;
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.comparator.CarManufacturerComparator_LowerCase;
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.Car;
+import de.uni_mannheim.informatik.dws.wdi.WDI_Project_T2.model.CarXMLReader;
+import de.uni_mannheim.informatik.dws.winter.matching.MatchingEngine;
+import de.uni_mannheim.informatik.dws.winter.matching.MatchingEvaluator;
+import de.uni_mannheim.informatik.dws.winter.matching.blockers.*;
+import de.uni_mannheim.informatik.dws.winter.matching.rules.LinearCombinationMatchingRule;
+import de.uni_mannheim.informatik.dws.winter.model.Correspondence;
+import de.uni_mannheim.informatik.dws.winter.model.HashedDataSet;
+import de.uni_mannheim.informatik.dws.winter.model.MatchingGoldStandard;
+import de.uni_mannheim.informatik.dws.winter.model.Performance;
+import de.uni_mannheim.informatik.dws.winter.model.defaultmodel.Attribute;
+import de.uni_mannheim.informatik.dws.winter.model.io.CSVCorrespondenceFormatter;
+import de.uni_mannheim.informatik.dws.winter.processing.Processable;
+import de.uni_mannheim.informatik.dws.winter.utils.WinterLogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.File;
+
+public class IR_using_linear_combination {
```

Comment:
Do we need this file at all? As far as I see it does nothing different than `IR_App`, but always picks the first x entries from offers instead of sampling.

Reply:
It was just for the first testing...